spec/pageflow/policies/folder_policy_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'spec_helper'
+
+module Pageflow
+  module Policies
+    describe FolderPolicy do
+      terms = {class: FolderPolicy,
+               subject: FactoryGirl.create(:folder),
+               name: 'folder'}
+      terms = terms.merge!(entity: terms[:subject].account)
+      describe :manage do
+        it_behaves_like 'a membership-based permission that',
+                        terms: terms,
+                        entity_type: :account,
+                        permission_type: :manage,
+                        minimum_required_role: 'publisher',
+                        maximum_forbidden_role: 'editor'
+      end
+    end
```

Comment:
Can we make the anonymous function take the environment of the shared example? Otherwise the lambda construct acts as a closure, and will for that reason not work as intended